src/ProjectOrigin.WalletSystem.Server/Models/CertificateViewModel.cs

```diff
@@ -1,11 +1,23 @@
 using System;
+using System.Collections.Generic;
+using System.Linq;
+using Google.Protobuf.WellKnownTypes;
 using ProjectOrigin.Register.V1;
 using ProjectOrigin.WalletSystem.V1;
 
 namespace ProjectOrigin.WalletSystem.Server.Models;
 
-public record CertificateViewModel(Guid Id, string Registry, long Quantity)
+public class CertificateViewModel
```

Comment:
How come you have changed it to a class, you already instantiate the attribute and slices with empty lists, so these could easily be filled without the set?